client/src/Components/Layout/Navbar/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Menu, Dropdown, Image, Typography } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+import PropTypes from 'prop-types';
+
+import MainButton from '../../Common/MainButton';
+import UserImage from '../../Common/UserImage';
+import NavTap from '../../Common/NavTap';
+import homedrop from '../../../assets/icons/homedrop.svg';
+import logout from '../../../assets/icons/logout.svg';
+import './style.css';
+
+const { Text } = Typography;
+
+const NavBar = ({ userName, isLogged, image }) => {
+  const menu = (
+    <Menu id="drop-down">
+      <Text className="user-name">{userName}</Text>
+      <Menu.Divider />
+
+      <Menu.Item key="1">
+        <Link to="!#">
+          <Image preview={false} src={homedrop} alt="home" />
+          <span className="drop-down-list-span">صفحة الروضة</span>
+        </Link>
+      </Menu.Item>
+      <Menu.Item key="2">
+        <Link to="!#">
+          <Image preview={false} src={logout} alt="logout" />
+          <span className="drop-down-list-span">تسجيل الخروج</span>
+        </Link>
+      </Menu.Item>
+    </Menu>
+  );
+
+  return (
+    <div className="navbar">
+      <div className="navbar-logo-div">
+        <h1>روضــتـــي</h1>
```

Comment:
Here you can use Title from antd instead of h1
```suggestion
       const { Title } = Typography;
       
        <Title level={1} className="navbar-logo-div">روضــتـــي</Title>
```